packages/web/package.json

```diff
@@ -50,7 +50,8 @@
 	},
 	"peerDependencies": {
 		"react": ">=15",
-		"react-dom": ">=15"
+		"react-dom": ">=15",
+		"hotkeys-js": "^3.8.5"
```

Comment:
@mohdashraf010897 Share a demo when `focusShortcuts` prop has not been used. It shouldn't require the `hotkeys-js`.

Reply:
https://www.loom.com/share/878a4f0a63dd43c99f8169ec52fcd87a